extensions/docker/src/docker-context-handler.spec.ts

```diff
@@ -210,11 +210,11 @@ describe('getContexts', () => {
     vi.spyOn(fs.promises, 'readdir').mockResolvedValue([
       {
         isDirectory: () => true,
-        name: 'fcde2b2edba56bf408601fb721fe9b5c338d10ee429ea04fae5511b68fbf8fb9',
+        name: '2c26b46b68ffc68ff99b453c1d30413413422d706483bfa0f98a5e886266e7ae',
```

Comment:
Not sure to get why this is required ?

Reply:
hello @jeffmaury it turns out that the test is fully broken (I don't know how it can work today)

the name 'foo' is encoded using sha256 into `2c26b46b68ffc68ff99b453c1d30413413422d706483bfa0f98a5e886266e7ae` and 'bar'
into `fcde2b2edba56bf408601fb721fe9b5c338d10ee429ea04fae5511b68fbf8fb9`

and how it's before this change means that foo and bar was the opposite values.

I suppose it's reusing some previous data in vite v2 while in vite3 the cleanup is really done so the order is not good (the test is broken)